schema/rdl_schema_0.1.json

```diff
@@ -594,39 +572,44 @@
 				"Observed"
 			]
 		},
-		"common_exp_category": {
+		"Category": {
```

Comment:
It is, it's referenced in `loss` and `vulnerability` as well

Reply:
Ah, I see. Nonetheless, I think it would be better to define the field in each object and have them all reference the same CSV codelist. That way the field description can be specific to their context. Also, I think that factoring out single fields might cause some problems for our documentation tooling.

Reply:
ah, okay yes that works, I'll make that change.